lib/skul/checker.js

```diff
@@ -82,4 +84,131 @@ module.exports.checkPermission = function(key) {
 	// Check permission
 	return this.permissionChecker[key].apply(this, checkerArguments);
 
-};
\ No newline at end of file
+};
+
+
+
+/**
+ * Check displayable keys
+ *
+ * @params {Object} options
```

Comment:
Ok, tu as fais comme les exemples sur lesquels tu t'es basé mais essaye de ne pas reproduire les erreurs que je fais moi. 😄 Voici le début d'un commentaire un peu mieux : 

```suggestion
 * @params {Object} context
 * @params {Object} context.options
```

La suite à faire par toi même...